### PR TITLE
[UI/UX:Forum] Clear Filters button wraps properly

### PR DESCRIPTION
**Pull request by ShashidharM0118**
### Please check if the PR fulfills these requirements:

* [ ] Tests for the changes have been added/updated (if possible)
* [ ] Screenshots are attached to Github PR if visual/UI changes were made

### What is the current behavior?
The "Clear Filters" button wraps poorly when displayed next to the "Filter" button
before

Fixes [Issue #11167](https://github.com/Submitty/Submitty/issues/11167)

### What is the new behavior?
The "Clear Filters" button now stays properly aligned
after

### Other information?
To test, go to the discussion forum, and check if the "Clear Filters" button remains properly aligned next to the "Filter" button without wrapping, even when the window is resized.

### CODE REVIEW

File: site/app/templates/forum/ForumBar.twig [hunk 1]
```diff
@@ -26,7 +26,7 @@
 				<div class="col-6">
 					<input type="text" placeholder="Search here..." name="search_content" id="search_content" onchange="$(this).val($(this).val().trim())" required aria-label="Forum Search Input Box" />
 				</div>
-				<div class="col-5">
+				<div class="col-5" style="display: flex; align-items: center; gap: 5px; flex-wrap: nowrap; width: auto;">
```
Comment (IDzyre) [inline site/app/templates/forum/ForumBar.twig:29]: CSS should go in a .css file to be linted

Comment (ShashidharM0118) [reply site/app/templates/forum/ForumBar.twig]: Moved inline CSS to a separate ForumBar.css file

**Review by zacharymnp — APPROVED** [review]
This works decently, although the overlap between the left-aligned buttons and the right-aligned buttons is not ideal.

**Review by williamjallen — APPROVED** [review]
The CSS for this part of the forum is not ideal, but this fix seems sufficient for now.

**Pull request merged**